diagnostic.md

```diff
@@ -6,19 +6,20 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+•by adding show in a resources only line of code
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+in the routes file
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+tells rails which db to migrate (pull data) from
```

Comment:
It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.